cmd/vic-machine/common/dns.go

```diff
@@ -0,0 +1,56 @@
+package common
+
+import (
+	"gopkg.in/urfave/cli.v1"
```

Comment:
go imports not formatted correctly - see [this thread](https://github.com/vmware/vic/pull/5373#discussion_r120218740).